src/audio/AudioResource.ts

```diff
@@ -251,3 +275,72 @@ export function createAudioResource<T>(
 		options.silencePaddingFrames ?? 5,
 	);
 }
+
+export function createFFMPEGResource<T>(
```

Comment:
Can we rename this to `createFFmpegResource` or `createFFmpegAudioResource`

Reply:
Yeah sure.